src/codex_plus/llm_execution_middleware.py

```
@@ -355,45 +400,286 @@ async def process_request(self, request, path: str):
+
+            if is_cerebras:
+                # 🔧 CEREBRAS PATH: Use httpx.stream (curl_cffi impersonation fails here)
+                logger.info("🌐 Using httpx.stream for Cerebras upstream")
+                try:
+                    debug_body = json.loads(body) if body else {}
+                    logger.info(
+                        f"📤 Sending to Cerebras: model={debug_body.get('model')}, "
+                        f"messages_count={len(debug_body.get('messages', []))}, "
+                        f"tools_count={len(debug_body.get('tools', []))}"
+                    )
+                    os.makedirs("/tmp/codex_plus/cerebras_debug", exist_ok=True)
```

Ruff (0.13.2)

Probable insecure usage of temporary file or directory: "/tmp/codex_plus/cerebras_debug"

(S108)